src/main/kotlin/uk/gov/gdx/datashare/service/DeathNotificationService.kt

```diff
@@ -98,16 +104,28 @@ class DeathNotificationService(
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @Schema(description = "Death notification")
 data class DeathNotificationDetails(
```

Comment:
There's probably a deeper future question for us to think about here which is does our service just pipe through the original entity, in which case calling it DeathNotificationDetails is right, or do we pipe through an event, with some citizen details from a variety of sources, in which case this structure doesn't feel quite right (maybe it's a `BaseEvent` with some standard citizen details and a couple of `DeathNotificationDetails` specific fields like `deathPlace`?)

Reply:
Yes think we should discuss and agree way forward.  Maybe think about another event and what the `BaseEvent` looks like